index.js

```diff
@@ -25,5 +25,6 @@ function isEmpty (val) {
   if ('number' == typeof val) return 0 === val;
   if (undefined !== val.length) return 0 === val.length;
   for (var key in val) if (has.call(val, key)) return false;
+  if (typeof val === 'boolean') return false;
```

Comment:
Can you move this line to above the `number` test, and swap it to be `if ('boolean' == typeof val)`. Just so that it matches that one before the array/object weirdness. Other than that looks great!